### DISCUSSION TITLE
Equivalent of DrawText

### DISCUSSION THREAD

**Discussion by AlexandreBazeaud** [SixLabors/ImageSharp.Drawing#208] [Q&A] [answered]
Hello,
since updating from Beta13 method `DrawText(DrawingOptions, string, Font, Color, PointF)` doesn't exist anymore, now i either have a TextOptions or a location, how can i have the same functionality than beta 13 ?

**Comment by JimBobSquarePants** [comment, marked as answer]
https://github.com/SixLabors/ImageSharp.Drawing/discussions/200

Please make sure you read existing discussions

**Reply by AlexandreBazeaud** [reply]
Sorry not familiar with discussions yet, thanks for the fast answer
